src/utils/trust/legal.entity.beneficial.owner.mapper.ts

```diff
@@ -123,18 +134,38 @@ const mapLegalEntityTrusteeFromSessionToPage = (
     registration_number: trustee.identification_registration_number,
     is_service_address_same_as_principal_address: trustee.is_service_address_same_as_principal_address,
     is_on_register_in_country_formed_in: trustee.is_on_register_in_country_formed_in,
+    ceasedDateDay: trustee.ceased_date_day,
+    ceasedDateMonth: trustee.ceased_date_month,
+    ceasedDateYear: trustee.ceased_date_year,
   } as Page.TrustLegalEntityForm;
 
-  if (trustee.type === RoleWithinTrustType.INTERESTED_PERSON){
+  let stillInvolvedInTrust: string;
+  switch (trustee.still_involved) {
```

Comment:
it might be better to use an enum

Reply:
Not sure an enum would fit this very well

Reply:
Saw the changes on https://github.com/companieshouse/overseas-entities-web/pull/1425 and have updated this.